contrib/generic_proxy/filters/network/test/integration_test.cc

```diff
@@ -22,6 +22,14 @@ namespace NetworkFilters {
 namespace GenericProxy {
 namespace {
 
+class GenericProxyIntegrationTest : public BaseIntegrationTest {
+public:
+  GenericProxyIntegrationTest(const std::string config_yaml)
+      : BaseIntegrationTest(Network::Address::IpVersion::v4, config_yaml) {
+    skip_tag_extraction_rule_check_ = true;
+  };
+};
+
```

Comment:
why this is necessary?

Reply:
I need to set skip_tag_extraction_rule_check_, otherwise the integration test will not pass